tests/test_visitInfo.py

```diff
@@ -136,7 +139,8 @@ def computeLstHA(data):
                                   22.2*degrees, 33.3*degrees, 0.444),
                               weather=Weather(2.2, 3.3, 44.4),
                               instrumentLabel="TestCameraTwo",
-                              id=123456
+                              id=123456,
+                              focusZ=-1.5,
```

Comment:
I'd suggest a number with a different absolute value. +1.5 and -1.5 might be hard to tell apart when debugging.